Source/WebCore/html/HTMLInputElement.cpp

```diff
@@ -1487,6 +1487,9 @@ bool HTMLInputElement::isOutOfRange() const
 
 bool HTMLInputElement::needsSuspensionCallback()
 {
+    if (!m_inputType)
+        return false;
```

Comment:
What is this change for? Change log doesn’t say.

Reply:
Same as above.